src/main/java/com/thevoxelbox/voxelsniper/brush/type/PullBrush.java

```diff
@@ -23,15 +23,26 @@
 
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
-        Double pinch = NumericParser.parseDouble(parameters[1]);
-        Double bubble = NumericParser.parseDouble(parameters[1]);
-        if (pinch == null || bubble == null) {
-            SnipeMessenger messenger = snipe.createMessenger();
-            messenger.sendMessage(ChatColor.RED + "Invalid brush parameters!");
-            return;
+        SnipeMessenger messenger = snipe.createMessenger();
+        String firstParameter = parameters[0];
+        if (firstParameter.equalsIgnoreCase("info")) {
+            messenger.sendMessage(ChatColor.GOLD + "Pull Brush Parameters:");
```

Comment:
How does one use this brush? `/b pull d 60` does nothing

Reply:
Just use ``/b pull 60``

Reply:
Oic, that works. It just doesn't respond anything. Also I think we should change letters that aren't `n` representing numbers.